iphone/Classes/TiPublicAPI.h

```diff
@@ -52,7 +52,7 @@
   }
 
 #define TI_PUBLIC_METHOD_END_ARGS(methodName, returnType) \
-  if (![@"void" isEqualToString:@"" #returnType]) {       \
+  if (![@"void" isEqualToString:@"" #returnType]) {      \
```

Comment:
clang-format is complaining (and rightfully so) that the line continuations don't line up